### PR TITLE
Update consul-helm enterprise docs for ACLs

### PR DESCRIPTION
**Pull request by lkysow**
If ACLs are added then slightly different commands are needed.

Note: needs #6180 merged first

### CODE REVIEW

File: website/source/docs/platform/k8s/service-sync.html.md [hunk 1]
```diff
@@ -53,7 +53,26 @@ syncCatalog:
...
+syncCatalog:
+  enabled: true
+  toConsul: true
+  toK8S: false
+```
+
+See the [Helm configuration](/docs/platform/k8s/helm.html#v-synccatalog)
 for more information.
 
 -> **Before installing,** please read the introduction paragraphs for the
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/service-sync.html.md]: This note feels a little bit out of place, and I'd love to find a way to put it above the installation instructions.

Comment (judithpatudith) [inline website/source/docs/platform/k8s/service-sync.html.md]:
```suggestion
~> **Before installing,** please read the introduction paragraphs for the
```

File: website/source/docs/platform/k8s/service-sync.html.md [hunk 2]
```diff
@@ -240,22 +261,36 @@ metadata:
 ## Consul to Kubernetes
 
 This syncs Consul services into first-class Kubernetes services.
-Each Consul service is synced to an
-[ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
-service in Kubernetes. The external name is configured to be the Consul
-DNS entry.
+An [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/service-sync.html.md]:
```suggestion
An [`ExternalName`](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
```

File: website/source/docs/platform/k8s/service-sync.html.md [hunk 3]
```diff
@@ -240,22 +261,36 @@ metadata:
...
+`Service` is created for each Consul service. The "external name" will be
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/service-sync.html.md]: Clarify who or what does the creating by putting this sentence into active voice?

File: website/source/docs/platform/k8s/service-sync.html.md [hunk 4]
```diff
@@ -240,22 +261,36 @@ metadata:
...
+metadata:
+  name: foo
+  ...
+spec:
+  externalName: foo.service.consul
+  type: ExternalName
+```
+
+With Consul To Kubernetes syncing enabled, DNS requests of the form `{consul-service-name}`
+will be serviced by Consul DNS. From a different namespace than where Consul
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/service-sync.html.md]:
```suggestion
will be serviced by Consul DNS. From a different Kubernetes namespace than where Consul
```

File: website/source/docs/platform/k8s/service-sync.html.md [hunk 5]
```diff
@@ -240,22 +261,36 @@ metadata:
...
+is deployed, the DNS request would need to be `{consul-service-name}.{consul-namespace}`.
 
-This enables external services to be discovered using native Kubernetes
-tooling. This can be used to ease software migration into or out of Kubernetes,
-across platforms, to and from hosted services, and more.
+-> **Note:** Consul to Kubernetes syncing **isn't required** if you've enabled [Consul DNS on Kubernetes](/docs/platform/k8s/dns.html)
+*and* all you need to do is address services in the form `{consul-service-name}.service.consul`, i.e. you don't need Kubernetes `Service` objects created.
 
 -> **Requires Consul DNS via CoreDNS in Kubernetes:** This feature requires that
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/service-sync.html.md]:
```suggestion
~> **Requires Consul DNS via CoreDNS in Kubernetes:** This feature requires that
```

File: website/source/docs/platform/k8s/dns.html.md [hunk 6]
```diff
@@ -9,19 +9,48 @@ description: |-
...
+[stub-domain configuration](https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#configure-stub-domain-and-upstream-dns-servers)
+if using KubeDNS or a [proxy configuration](https://coredns.io/plugins/proxy/) if using CoreDNS.
 
-The stub-domain configuration must point to a static IP of a DNS resolver.
-The [Helm chart](/docs/platform/k8s/helm.html) creates a `consul-dns` service
-by default that exports Consul DNS. The cluster IP of this service can be used
-to configure a stub-domain with kube-dns. While the `kube-dns` configuration
-lives in the `kube-system` namespace, the IP just has to be routable so the
-service can live in a different namespace.
+Once configured, DNS requests in the form `{consul-service-name}.service.consul` will
```
[the remark this answers is not in the document]
Comment (lkysow) [reply website/source/docs/platform/k8s/dns.html.md]: Not sure, I'll check.

[the remark this answers is not in the document]
Comment (lkysow) [reply website/source/docs/platform/k8s/dns.html.md]: I think `<>` is more standard. Will update.

**Pull request merged**